homeassistant/components/foscam/camera.py

```diff
@@ -27,7 +27,7 @@
     vol.Required(CONF_USERNAME): cv.string,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
-    vol.Optional(CONF_RTSP_PORT, default=None): vol.Any(None, cv.port)
+    vol.Optional(CONF_RTSP_PORT): vol.Any(None, cv.port)
```

Comment:
Remove `vol.Any` and just use `cv.port`.

Reply:
whoops. I had that there from when I was setting None as the default. Updated.

Reply:
Sorry, I'm new here 😄